src/qt/rpcconsole.cpp

```diff
@@ -72,6 +72,8 @@ const QStringList historyFilter = QStringList()
     << "importmulti"
     << "signmessagewithprivkey"
     << "signrawtransaction"
+    << "signrawtransactionwithwallet"
```

Comment:
`signrawtransactionwithwallet` doesn't handle private keys, so I don't think it needs to be included here.

Reply:
Done.